vllm/v1/attention/backends/flash_attn.py

```diff
@@ -213,10 +217,13 @@ def unified_flash_attention(
         block_table=attn_metadata.block_table,
         softcap=logits_soft_cap,
     )
-    return output.view(num_tokens, hidden_size)
+    output = output.view(num_actual_tokens, -1)
+    # TODO(woosuk): Optimize this.
+    out[:num_actual_tokens].copy_(output, non_blocking=True)
```

Comment:
`non_blocking` seems unnecessary. this is cuda tensor, it should be non_blocking by default.

Reply:
Good catch. Fixed.